index.html

```diff
@@ -4,9 +4,42 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Raj</title>
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,400;0,600;0,700;1,100;1,300;1,400;1,600;1,700&display=swap" rel="stylesheet">
+
+    <link rel="stylesheet" href="css/style.css">
+    
 </head>
 <body>
-    
+    <header>
+        <nav class="menu">
+            <div>
+                <a href="#" class="logo">My Logo</a>
+            </div>
+            <div class="menu-icon">
+                <img src="/img/Icon - Menu.png" alt="">
```

Comment:
Kindly make sure there is no space on the name of your images that's why you can see they are not displayed on your project.
`hint`: `You can separate the name with an hyphen or underscore not a generic space` 
![Screenshot from 2022-09-21 19-06-56](https://user-images.githubusercontent.com/97100091/191567658-0f6d2e21-4263-46b8-ad35-6e690abe8471.png)